### ISSUE TITLE
Option to change ticket status when replying

### ISSUE THREAD

**Issue by theresabugforsure** [freescout-helpdesk/freescout#2147]
A useful feature found in a lot of ticketing systems is to be able to close a ticket when posting a final reply. I'm not sure if this is possible with Freescout, but it would reduce a multi-step process to a single step. Maybe add the option "Send and Close" to the send button or have a checkbox next to it to close the ticket.

**Comment by ilyakonovalenko** [comment]
You can select new status when replying. It is already there...

**Comment by MarcoWel** [comment]
It is there, but is there a way to make the selection permanent? Also regarding what happens when sending a reply (stay or proceed to next ticket). Currently, opening a new tab seems to fall back to the default setting here.

**Comment by ilyakonovalenko** [comment]
You can set default status which will be used on reply. It is per mailbox, but I think it can be changed in any way via modules.

**Comment by MarcoWel** [comment]
Yes, but every support agent has a different preference which might change as well depending on the situation.
A mailbox global default value is not practical here.

**Comment by ilyakonovalenko** [comment]
I agree there is much room for improvement. It is good to have a way to set such settings per user, not per mailbox. There also should be an option to keep the status the same when replying. Not sure if @freescout-helpdesk would like to change it.

For now it should be possible to change the behaviour via modules.

**Comment by theresabugforsure** [comment]
I don't see how a default status is a useful feature. If an agent sets their default reply status to Closed, that means it would close every ticket they reply to if they forget to change that before replying. Or am I not understanding?

**Comment by ilyakonovalenko** [comment]
I believe the best way is to select new status manually on reply (status should not be changed by default on reply). Changing status on reply to some default status is not good solution. Another button like "Reply & Close" should be also fine.

**Comment by freescout-helpdesk** [comment]
There is no plans to add this kind of feature to the core of the app.
